deepcell/model_zoo/featurenet.py

```diff
@@ -446,7 +453,39 @@ def bn_feature_net_3D(receptive_field=61,
                     kernel_initializer=init, padding='valid',
                     kernel_regularizer=l2(reg))(x[-1]))
     x.append(BatchNormalization(axis=channel_axis)(x[-1]))
-    x.append(Activation('relu')(x[-1]))
+    feature = Activation('relu')(x[-1])
+
+    def __merge_temporal_features(feature, mode='conv', residual=False, n_filters=256,
+                                  n_frames=3, padding=True, temporal_kernel_size=3):
+        if mode is None:
+            return feature
+        if mode.lower() == 'conv':
```

Comment:
I suggest casting mode as a string. If a user unwisely uses `1` as the mode, then we will get an `AttributeError`.  We can fix this by simply casting in the begginning, with `mode = str(mode).lower()`.  Then we can just use simple comparisons, `if mode == 'conv':`

Reply:
Also, when you update this, please update the PR from master.

Reply:
Done!

Reply:
Looks like there is some small linting issue:

```
deepcell-tf/deepcell/model_zoo/featurenet.py:462:1: W293 blank line contains whitespace
```

Reply:
Oops, fixed and ran tests -- should work now?